build/tradable.js

```diff
@@ -0,0 +1,26 @@
+const builtUntradable = ['Warframe', 'Throwing', 'Shotgun', 'Rifle', 'Pistol', 'Melee', 'Sword And Shield']
+const tradableConditions = (i) => !(builtUntradable.includes(i.type) && i.name.match(/Prime/ig))
+
+const tradableTypes = ['Upgrades', 'Arcane', 'Fish', 'Focus Lens', 'Relic', 'Secondary Mod', 'Shotgun Mod',
+  'Warframe Mod', 'Companion Mod', 'Archwing Mod', 'K-Drive Mod', 'Melee Mod', 'Arch-Melee Mod', 'Necramech Mod',
+  'Cut Gem', 'Captura', 'Primary Mod', 'Sentinel Mod', 'Kubrow Mod', 'Kavat Mod']
+const untradableTypes = ['Skin', 'Medallion', 'Key', 'Extractor', 'Pets', 'Ship Decoration',
+  'Glyph', 'Sigil', 'Fur Color', 'Syandana', 'Fur Pattern', 'Color Palette', 'Node', 'Exalted Weapon']
+const tradableRegex = /(Prime|Vandal|Wraith\w|\wWraith|Rakta|Synoid|Sancti|Vaykor|Telos|Secura|Ayatan|Prisma|DamagedMech)(?!Derelict)/i
+const untradableRegex = /(Glyph|Mandachord|Greater.*Lens|Sugatra|\[|SentinelWeapons|Toroid|Bait|([A-Za-z]+ (Relic)))|Umbral|Sacrificial/i
+
+/**
+ * Check if an item is tradable
+ * @param {module:warframe-items.Item} item
+ * @returns {boolean}
+ */
+module.exports = (item) => {
```

Comment:
What do you think about naming this function instead of just using an arrow function? Sometimes it helps debugging.
```suggestion
module.exports = function isTradable(item) {
```

Reply:
It's literally in it's own file...

Reply:
Yeah i guess it doesn't matter much in this case